catebi_api/catebi_endpoints.py

```diff
@@ -0,0 +1,16 @@
+from functools import lru_cache
+
+
+class CatebiEndpoints:
+
+    @classmethod
+    @lru_cache()
+    def save_reaction(cls) -> str:
+        """Return save reaction endpoint URL. """
+        return "https://api.catebi.ge/api/Freegan/SaveReaction"
```

Comment:
i suggest to move main part of the url to the .env
so, https://api.catebi.ge/api/ should be in the .env file

Reply:
Good idea